dist.ini

```diff
@@ -0,0 +1,80 @@
+name    = Pod-Usage
+author  = Marek Rouchal <marekr@cpan.org>
+license = Perl_5
+copyright_holder = Marek Rouchal <marekr@cpan.org>
+copyright_year   = 2013
+
+[ReadmeAnyFromPod / Markdown_Readme]
+type = gfm
+source_filename = lib/Pod/Usage.pm
+filename = README.md
+location = root
+
+[Regenerate::AfterReleasers]
+plugin = Markdown_Readme
+
+[@Starter::Git]
+revision = 4
+managed_versions = 1
+installer = MakeMaker::Awesome
+RewriteVersion.global = 1
+NextRelease.format = %-9v %{yyyy-MM-dd}d
+regenerate = Makefile.PL
+regenerate = META.json
+regenerate = README.md
+regenerate = LICENSE
+regenerate = t/00-report-prereqs.t
+regenerate = t/00-report-prereqs.dd
+ExecDir.dir = script
+
+;MakeMaker::Awesome.exe_file = scripts/pod2usage
+MakeMaker::Awesome.WriteMakefile_arg[0] = EXE_FILES => MY::exe()
+MakeMaker::Awesome.WriteMakefile_arg[1] = PL_FILES => { MY::pod2usage() . '.PL', MY::pod2usage() }
+MakeMaker::Awesome.WriteMakefile_arg[2] = INSTALLDIRS  => ($] < 5.012 ? 'perl' : 'site')
+MakeMaker::Awesome.WriteMakefile_arg[3] = clean => { FILES => join( ' ', @{MY::exe()}) }
+MakeMaker::Awesome.delimiter =  |
+MakeMaker::Awesome.footer[00] = |# <custom footer>
+MakeMaker::Awesome.footer[01] = |package MY; use strict; use warnings;
```

Comment:
Personally I think this using MY and subroutines is far overcomplicated when there is only one script, I would prefer setting PL_FILES etc with inline conditions as needed. But no other objections.

Reply:
I was just trying to preserve the existing customized logic in the Makefile.PL
I would have also preferred not maintaining this part, but I'm trying to avoid regressions when bumping to Dsit::Zilla.
We can definitively improve this by some later commits IMO